Library/Formula/moc.rb

```diff
@@ -37,8 +34,12 @@ class Moc < Formula
   depends_on "libmagic" => :optional
   depends_on "homebrew/dupes/ncurses" => :optional
 
+  # Backport r2779: Adapt to FFmpeg/LibAV's audioconvert.h rename (commit 5980f5dd).
+  # Necessary for building against FFmpeg 3.0. See http://moc.daper.net/node/1496.
+  patch :DATA
```

Comment:
Also, if this is already done upstream you'll want to wrap it in a `stable do` block with the stable URL/SHA256 so it isn't applied to the `head` build as well.